### PR TITLE
No issue: Version bump to 3.6 and update changelog

### PR DESCRIPTION
**Pull request by liuche**
Version bump + changelog to bring master up to 3.6 version.

### Pull Request checklist

- [ ] This PR includes thorough **tests** or an explanation of why it does not
- [X] This PR includes a **CHANGELOG entry** or does not need one
- [ ] I have considered adding **QA labels** on the associated issue (not this PR; `qa-ready` or `qa-denied`)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,13 +10,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
 ### Fixed
-- Bug where you could not use remote back button to exit YouTube if it was visited from Pocket (#1584)
-- Fixed 4K YT bug (#277)
-- Backing from the navigation overlay when on the first site in your backstack will now properly exit the app (#1916)
-- Fixed Youtube back bug (#1939)
-- Workaround for "grey screen" when returning to YouTube from Amazon homescreen (#1865)
+
+# [3.6-RO] 2019-03-26
+#*Released to Fire TV 4K, staged roll-out*
```
Comment (dnarcese) [inline CHANGELOG.md]: The # on this line should be on the line above

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,13 +10,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+### Fixed
 - Bug that prevented the toolbar back button from working on YouTube (#1927)
 
+## [3.5-A] - 2019-03-26
+*Staged rollout of 3.5, no change from v3.5-RO. Released to all devices except Fire TV 4K*
+
+
```
Comment (dnarcese) [inline CHANGELOG.md]: Remove this blank line?

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,13 +10,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
-- Backing from the navigation overlay when on the first site in your backstack will now properly exit the app (#1916)
-- Fixed Youtube back bug (#1939)
-- Workaround for "grey screen" when returning to YouTube from Amazon homescreen (#1865)
+
+# [3.6-RO] 2019-03-26
+#*Released to Fire TV 4K, staged roll-out*
+### Fixed
 - Bug that prevented the toolbar back button from working on YouTube (#1927)
 
+## [3.5-A] - 2019-03-26
```
Comment (dnarcese) [inline CHANGELOG.md]: There is no link for this release, so we should remove the square brackets.
Same thing for the 3.4-B release!

**Comment by liuche** [comment]
Moved this to #1985

**Pull request closed without merging**